### ISSUE TITLE
bug: build fail in 'verbose' model since 1.10 version

### ISSUE THREAD

**Issue by jacobke** [dominique-mueller/simple-progress-webpack-plugin#27]
Hi, this is one really nice plugin. When i configure to `verbose` model, it compiles failed like: 

```
myproject/node_modules/simple-progress-webpack-plugin/logger/verbose-logger.js:17
        chalk.enabled = options.color;
        ^

ReferenceError: chalk is not defined
```

refer to #20, it seems that PR missed to import `chark`  in `verbose-logger.js`

```
// existed in other modes
const chalk = require( 'chalk' );
```

could you fix this, please? 😄

**Comment by dominique-mueller** [comment]
Yep, seems you're right, the import it also missing in the expanded logger. I'll look at it today :)

**Comment by dominique-mueller** [comment]
Made a bug fix, and published version `1.1.1` to npm. Have fun!
